### ISSUE TITLE
Usage with gulp.src passthrough

### ISSUE THREAD

**Issue by ivogabe** [floridoo/gulp-sourcemaps#147]
In recent versions of gulp you can use `gulp.src` to add extra files to the stream. For example you can first compile your sources and then add some libraries before you concatenate them. As far as I can see, this can't be used with gulp-sourcemaps, since the new files won't have source maps. Adding init after the second `gulp.src` call will overwrite the existing source maps. What do you think of an extra option, like `passthrough`:
```javascript
return gulp.src('...')
  .pipe(sourcemaps.init())
  .pipe(typescript())
  .pipe(gulp.src('...', { passthrough: true }))
  .pipe(sourcemaps.init({ passthrough: true })) // Only add source maps to new files
  .pipe(concat('scripts.js'))
  .pipe(sourcemaps.write())
  .pipe(gulp.dest('release'))
```

**Comment by floridoo** [comment]
Fixed by ad99c9637187035ea59ad006d2d0794e1b198710 (without an additional option)

**Comment by ivogabe** [comment]
👍
